<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" total="13" passed="13">
  <reporter-output>
  </reporter-output>
  <suite name="Automation Project Test Suite" duration-ms="140" started-at="2013-05-11T16:42:14Z" finished-at="2013-05-11T16:42:14Z">
    <groups>
      <group name="initQAEnv">
        <method signature="InitializationTests_TestSet.isQAEnvOK()[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.InitializationTests_TestSet@134a7d8]" name="isQAEnvOK" class="com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.InitializationTests_TestSet"/>
      </group> <!-- initQAEnv -->
      <group name="Abstract.TC.basic">
        <method signature="AcceptanceTests_Feature_AbstractTC.isAbstractTCworks(com.organization.automation.project.spice.mix.testng.test.AbstractTestCase)[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.AcceptanceTests_Feature_AbstractTC@1d85f79]" name="isAbstractTCworks" class="com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.AcceptanceTests_Feature_AbstractTC"/>
      </group> <!-- Abstract.TC.basic -->
      <group name="initLocalEnv">
        <method signature="InitializationTests_TestSet.isLocalTAFEnvOK()[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.InitializationTests_TestSet@134a7d8]" name="isLocalTAFEnvOK" class="com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.InitializationTests_TestSet"/>
      </group> <!-- initLocalEnv -->
      <group name="CustomAssertTests">
        <method signature="CustomAssertUnitTests.testBigIntegerAssert_bigNumbersCase()[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="testBigIntegerAssert_bigNumbersCase" class="com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests"/>
        <method signature="CustomAssertUnitTests.testBigIntegerAssert_bigNumbersEqualObjectCase()[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="testBigIntegerAssert_bigNumbersEqualObjectCase" class="com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests"/>
        <method signature="CustomAssertUnitTests.testBigIntegerAssert_bigNumbersLessCase()[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="testBigIntegerAssert_bigNumbersLessCase" class="com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests"/>
        <method signature="CustomAssertUnitTests.testBigIntegerAssert_bigNumbersMoreCase()[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="testBigIntegerAssert_bigNumbersMoreCase" class="com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests"/>
        <method signature="CustomAssertUnitTests.testBigIntegerAssert_mixNumbersLessCase()[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="testBigIntegerAssert_mixNumbersLessCase" class="com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests"/>
        <method signature="CustomAssertUnitTests.testBigIntegerAssert_mixNumbersMoreCase()[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="testBigIntegerAssert_mixNumbersMoreCase" class="com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests"/>
        <method signature="CustomAssertUnitTests.testBigIntegerAssert_smallNumbersCase()[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="testBigIntegerAssert_smallNumbersCase" class="com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests"/>
        <method signature="CustomAssertUnitTests.testBigIntegerAssert_smallNumbersLessCase()[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="testBigIntegerAssert_smallNumbersLessCase" class="com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests"/>
        <method signature="CustomAssertUnitTests.testBigIntegerAssert_smallNumbersMoreCase()[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="testBigIntegerAssert_smallNumbersMoreCase" class="com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests"/>
        <method signature="CustomAssertUnitTests.testBigIntegerAssert_smallNumbersNegativeCase()[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="testBigIntegerAssert_smallNumbersNegativeCase" class="com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests"/>
      </group> <!-- CustomAssertTests -->
      <group name="AbstractTCs">
        <method signature="AcceptanceTests_Feature_AbstractTC.isAbstractTCworks(com.organization.automation.project.spice.mix.testng.test.AbstractTestCase)[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.AcceptanceTests_Feature_AbstractTC@1d85f79]" name="isAbstractTCworks" class="com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.AcceptanceTests_Feature_AbstractTC"/>
      </group> <!-- AbstractTCs -->
      <group name="InitializationTests">
        <method signature="InitializationTests_TestSet.isLocalTAFEnvOK()[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.InitializationTests_TestSet@134a7d8]" name="isLocalTAFEnvOK" class="com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.InitializationTests_TestSet"/>
        <method signature="InitializationTests_TestSet.isQAEnvOK()[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.InitializationTests_TestSet@134a7d8]" name="isQAEnvOK" class="com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.InitializationTests_TestSet"/>
      </group> <!-- InitializationTests -->
    </groups>
    <test name="AcceptanceTests" duration-ms="140" started-at="2013-05-11T16:42:14Z" finished-at="2013-05-11T16:42:14Z">
      <class name="com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.AcceptanceTests_Feature_AbstractTC">
        <test-method status="PASS" signature="setUpSet()[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.AcceptanceTests_Feature_AbstractTC@1d85f79]" name="setUpSet" is-config="true" duration-ms="16" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpSet -->
        <test-method status="PASS" signature="setUpTest()[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.AcceptanceTests_Feature_AbstractTC@1d85f79]" name="setUpTest" is-config="true" duration-ms="0" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTest -->
        <test-method status="PASS" signature="isAbstractTCworks(com.organization.automation.project.spice.mix.testng.test.AbstractTestCase)[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.AcceptanceTests_Feature_AbstractTC@1d85f79]" name="isAbstractTCworks" duration-ms="31" depends-on-groups="InitializationTests" started-at="2013-05-11T19:42:14Z" data-provider="Abstract.TC" finished-at="2013-05-11T19:42:14Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[com.organization.automation.project.spice.mix.testng.test.AbstractTestCase@1c56c60]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isAbstractTCworks -->
        <test-method status="PASS" signature="tearDownTest()[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.AcceptanceTests_Feature_AbstractTC@1d85f79]" name="tearDownTest" is-config="true" duration-ms="0" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTest -->
        <test-method status="PASS" signature="tearDownSet()[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.AcceptanceTests_Feature_AbstractTC@1d85f79]" name="tearDownSet" is-config="true" duration-ms="0" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownSet -->
      </class> <!-- com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.AcceptanceTests_Feature_AbstractTC -->
      <class name="com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests">
        <test-method status="PASS" signature="setUpAllTests(java.lang.String)[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="setUpAllTests" is-config="true" duration-ms="0" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Unit Tests on Custom Assert]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpAllTests -->
        <test-method status="PASS" signature="testBigIntegerAssert_bigNumbersCase()[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="testBigIntegerAssert_bigNumbersCase" duration-ms="0" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testBigIntegerAssert_bigNumbersCase -->
        <test-method status="PASS" signature="testBigIntegerAssert_mixNumbersLessCase()[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="testBigIntegerAssert_mixNumbersLessCase" duration-ms="0" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Actual number should be equal expected. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Actual number should be equal expected. expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssert.assertEquals(CustomAssert.java:39)
	at com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests.testBigIntegerAssert_mixNumbersLessCase(CustomAssertUnitTests.java:121)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:73)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:94)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testBigIntegerAssert_mixNumbersLessCase -->
        <test-method status="PASS" signature="testBigIntegerAssert_smallNumbersLessCase()[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="testBigIntegerAssert_smallNumbersLessCase" duration-ms="0" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Actual number should be equal expected. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Actual number should be equal expected. expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssert.assertEquals(CustomAssert.java:39)
	at com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests.testBigIntegerAssert_smallNumbersLessCase(CustomAssertUnitTests.java:89)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:73)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:94)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testBigIntegerAssert_smallNumbersLessCase -->
        <test-method status="PASS" signature="testBigIntegerAssert_bigNumbersEqualObjectCase()[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="testBigIntegerAssert_bigNumbersEqualObjectCase" duration-ms="0" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testBigIntegerAssert_bigNumbersEqualObjectCase -->
        <test-method status="PASS" signature="testBigIntegerAssert_bigNumbersMoreCase()[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="testBigIntegerAssert_bigNumbersMoreCase" duration-ms="0" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Actual number should be equal expected. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Actual number should be equal expected. expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssert.assertEquals(CustomAssert.java:39)
	at com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests.testBigIntegerAssert_bigNumbersMoreCase(CustomAssertUnitTests.java:97)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:73)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:94)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testBigIntegerAssert_bigNumbersMoreCase -->
        <test-method status="PASS" signature="testBigIntegerAssert_smallNumbersCase()[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="testBigIntegerAssert_smallNumbersCase" duration-ms="0" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testBigIntegerAssert_smallNumbersCase -->
        <test-method status="PASS" signature="testBigIntegerAssert_mixNumbersMoreCase()[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="testBigIntegerAssert_mixNumbersMoreCase" duration-ms="0" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Actual number should be equal expected. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Actual number should be equal expected. expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssert.assertEquals(CustomAssert.java:39)
	at com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests.testBigIntegerAssert_mixNumbersMoreCase(CustomAssertUnitTests.java:113)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:73)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:94)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testBigIntegerAssert_mixNumbersMoreCase -->
        <test-method status="PASS" signature="testBigIntegerAssert_bigNumbersLessCase()[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="testBigIntegerAssert_bigNumbersLessCase" duration-ms="0" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Actual number should be equal expected. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Actual number should be equal expected. expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssert.assertEquals(CustomAssert.java:39)
	at com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests.testBigIntegerAssert_bigNumbersLessCase(CustomAssertUnitTests.java:105)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:73)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:94)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testBigIntegerAssert_bigNumbersLessCase -->
        <test-method status="PASS" signature="testBigIntegerAssert_smallNumbersNegativeCase()[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="testBigIntegerAssert_smallNumbersNegativeCase" duration-ms="0" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Actual number should be equal expected. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Actual number should be equal expected. expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssert.assertEquals(CustomAssert.java:39)
	at com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests.testBigIntegerAssert_smallNumbersNegativeCase(CustomAssertUnitTests.java:129)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:73)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:94)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testBigIntegerAssert_smallNumbersNegativeCase -->
        <test-method status="PASS" signature="testBigIntegerAssert_smallNumbersMoreCase()[pri:0, instance:com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests@a17083]" name="testBigIntegerAssert_smallNumbersMoreCase" duration-ms="0" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Actual number should be equal expected. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Actual number should be equal expected. expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssert.assertEquals(CustomAssert.java:39)
	at com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests.testBigIntegerAssert_smallNumbersMoreCase(CustomAssertUnitTests.java:81)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:73)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:94)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testBigIntegerAssert_smallNumbersMoreCase -->
      </class> <!-- com.organization.automation.project.spice.mix.testng.extension.assertions.CustomAssertUnitTests -->
      <class name="com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.InitializationTests_TestSet">
        <test-method status="PASS" signature="setUpSuite(java.lang.String)[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.InitializationTests_TestSet@134a7d8]" name="setUpSuite" is-config="true" duration-ms="16" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Automation Project]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpSuite -->
        <test-method status="PASS" signature="setUpSet(java.lang.String)[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.InitializationTests_TestSet@134a7d8]" name="setUpSet" is-config="true" duration-ms="0" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Initialization Tests]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpSet -->
        <test-method status="PASS" signature="isLocalTAFEnvOK()[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.InitializationTests_TestSet@134a7d8]" name="isLocalTAFEnvOK" duration-ms="15" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isLocalTAFEnvOK -->
        <test-method status="PASS" signature="setUpTest()[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.InitializationTests_TestSet@134a7d8]" name="setUpTest" is-config="true" duration-ms="0" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTest -->
        <test-method status="PASS" signature="setUpTest()[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.InitializationTests_TestSet@134a7d8]" name="setUpTest" is-config="true" duration-ms="16" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTest -->
        <test-method status="PASS" signature="tearDownTest()[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.InitializationTests_TestSet@134a7d8]" name="tearDownTest" is-config="true" duration-ms="0" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTest -->
        <test-method status="PASS" signature="tearDownSet()[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.InitializationTests_TestSet@134a7d8]" name="tearDownSet" is-config="true" duration-ms="15" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownSet -->
        <test-method status="PASS" signature="tearDownTest()[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.InitializationTests_TestSet@134a7d8]" name="tearDownTest" is-config="true" duration-ms="0" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTest -->
        <test-method status="PASS" signature="isQAEnvOK()[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.InitializationTests_TestSet@134a7d8]" name="isQAEnvOK" duration-ms="0" depends-on-groups="initLocalEnv" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isQAEnvOK -->
        <test-method status="PASS" signature="tearDownSuite()[pri:0, instance:com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.InitializationTests_TestSet@134a7d8]" name="tearDownSuite" is-config="true" duration-ms="0" started-at="2013-05-11T19:42:14Z" finished-at="2013-05-11T19:42:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownSuite -->
      </class> <!-- com.organization.automation.project.spice.mix.testng.feature.abstract_tc.tests.InitializationTests_TestSet -->
    </test> <!-- AcceptanceTests -->
  </suite> <!-- Automation Project Test Suite -->
</testng-results>
